### PR TITLE
Implement email router and FAQ suggestions for the contact form

### PR DESCRIPTION
**Pull request by GRardB**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -16,6 +16,13 @@ EMAIL_SERVER_PASSWORD=
 EMAIL_SERVER_HOST=
 EMAIL_SERVER_PORT=
 EMAIL_FROM=
+EMAIL_TO=
+EMAIL_TO_AVOCADO=
+EMAIL_TO_BANANA=
+EMAIL_TO_CARROT=
+EMAIL_TO_CHEATSHEET=
+EMAIL_TO_SWEET_POTATO=
+EMAIL_TO_WATERMELON=
```
Comment (GRardB) [inline .env.example:27]: I figured it'd be a good idea to keep people's emails out of the repo (also good for changing them if we need to) 😁

File: src/components/forms/contactUs.tsx [hunk 2]
```diff
@@ -129,8 +131,17 @@ const ContactUsForm: React.FC = () => {
...
           />
         </div>
+        {suggestions.map((suggestion, i) => (
+          <div
+            className="text-left border-l-8 border-l-yellow bg-white p-4"
+            key={i}
+          >
+            {suggestion}
+          </div>
+        ))}
```
Comment (GRardB) [inline src/components/forms/contactUs.tsx:141]: Contact_Us___Vegan_Hacktivists


Cursor_and_Contact_Us___Vegan_Hacktivists

File: src/lib/mail/index.ts [hunk 3]
```diff
@@ -13,12 +16,11 @@ const mg = mailgun.client({
...
 
-interface Email {
-  to: string;
-  from: string;
-  subject: string;
-  html: string;
-}
+interface Email
+  extends Required<
+      Pick<MailgunMessageData, 'to' | 'from' | 'subject' | 'html'>
```
Comment (GRardB) [inline src/lib/mail/index.ts]: I can revert this change if you want. It just felt more Typescript-y to me, but it's ultimately not necessary lol.

File: src/lib/mail/router/index.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { doesContainKeywords } from 'lib/helpers/strings';
+
+export const OUR_EMAIL = process.env.EMAIL_TO!;
+
+const projectSpecificEmails: Record<string, (string | string[])[]> = {
+  [process.env.EMAIL_TO_AVOCADO!]: [],
```
Comment (GRardB) [inline src/lib/mail/router/index.ts:15]: I wasn't sure what to put here 😅

**Comment by JoaquinTrinanes** [comment]
I just merged `main` and solved conflicts, I'll have to check that nothing breaks :)

**Review by Headscracher — APPROVED** [review]
Looking good. Please talk to me about the env variable first, before merging ;)

**Pull request merged**